lib/src/server/server.dart

```diff
@@ -111,8 +116,8 @@ class Server {
       // timeout.
       connection.incomingStreams.listen((stream) {
         handler = serveStream_(stream);
-      }, onError: (error) {
-        print('Connection error: $error');
+      }, onError: (error, StackTrace stackTrace) {
+        _log.severe('Connection error: ', error, stackTrace);
```

Comment:
While I don't disagree with this change, I am a bit worried about dragging in another dependency.
Can we do the logging part in another PR?

Reply:
done https://github.com/grpc/grpc-dart/pull/252